qutebrowser/utils/urlutils.py

```diff
@@ -176,7 +179,7 @@ def fuzzy_url(urlstr, cwd=None, relative=False, do_search=True):
 
     if path is not None:
         url = QUrl.fromLocalFile(path)
-    elif (not do_search) or is_url(urlstr):
+    elif not force_search and (not do_search or is_url(urlstr)):
```

Comment:
I'd put the force_search check as first because is pointless to check if the input is an url if we are going to open it as a search anyway.
Do you prefer the order you proposed in the condition nonetheless?